applications/server/views/log_test/log_report.html

```diff
@@ -0,0 +1,19 @@
+{{extend 'layout.html'}}
+<script src="/server/static/js/dygraph-combined-dev.js"></script>
+<h1>Visualization Test</h1>
+<center>
+    <div id="graphdiv" style="width: 600px; height: 400px"></div>
+</center>
+<script type="text/javascript">
+    g = new Dygraph(
+        document.getElementById("graphdiv"),
+        "{{=URL('static', 'temper.csv')}}",
```

Comment:
In line 31 of log_test.py, I write temper,csv file in static folder 
did I answer you question? 
Or do you want me check if there is the same name file in static folder?